docs/modelserving/autoscaling/autoscaling.md

```diff
@@ -1,4 +1,6 @@
 # Autoscale InferenceService with inference workload
+- The examples below depend on Knative. You need to implement HPA yourself without Knative.
+- To disable the HPA created by KServe, set serving.kserve.io/autoscalerClass: "external" in the InferenceService annotations.
```

Comment:
serving.kserve.io/autoscalerClass: "external" to `serving.kserve.io/autoscalerClass: "external"` to align the format with other contents.